js/util.js

```diff
@@ -0,0 +1,21 @@
+let randomInteger = function (min, max) {
```

Comment:
```suggestion
const randomInteger = function (min, max) {
```

https://up.htmlacademy.ru/javascript/22/criteries#b20

> Модуль не должен экспортировать переменную, значение которой может измениться в будущем.